src/MoBi.Presentation/Presenter/ObjectPathCreatorAtTransport.cs

```diff
@@ -48,7 +48,7 @@ public override ReferenceDTO CreatePathFromParameterDummy(ObjectBaseDTO objectBa
 
          return new ReferenceDTO
          {
-            Path = CreateFormulaUsablePathFrom(new[] { firstPathElement, dtoDummyParameter.ModelParentName, dtoDummyParameter.Name }, dtoDummyParameter.Parameter)
+            Path = CreateFormulaUsablePathFrom(new[] { firstPathElement, ObjectPathKeywords.MOLECULE, dtoDummyParameter.Name }, dtoDummyParameter.Parameter)
```

Comment:
will it also fix the "Adding molecule **amount**" case?
(I'm not sure, but I assume it should work the same way - just asked the question in the issue: https://github.com/Open-Systems-Pharmacology/MoBi/issues/1619#issuecomment-2491292797)